velociraptor/catalogue/registration.py

```diff
@@ -1442,7 +1443,18 @@ def registration_spherical_overdensities(
     else:
         raise RegistrationDoesNotMatchError
 
+def registration_bgpart_masses(
+    field_path: str, unit_system: VelociraptorUnits
+) -> (unyt.Unit, str, str):
+    """
+    Registers the halo mass contributed by background particle interlopers.
+    """
+    if field_path == "Mass_interloper":
+        return unit_system.mass, "Mass from background particles", "bgpart_masses"
+    else:
+        raise RegistrationDoesNotMatchError
 
+    
```

Comment:
Why is this separate from other masses?

Reply:
this should be integrated with the others now

Reply:
:+1: